Components/WideImage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView, Image, StatusBar, FlatList, TouchableOpacity, TouchableNativeFeedback,
+  Button,
+  Platform } from 'react-native';
+import { DefaultTheme, Headline, Paragraph, TextInput,
+  Appbar, AppbarAction, AppbarBackAction, AppbarContent, AppbarHeader, 
+  Card, CardContent,
+  // Button,
+  ListAccordion, Divider,
+  ListSection, withTheme,
+  Toolbar, ToolbarBackAction, ToolbarContent, ToolbarAction, 
+  FAB, DrawerSection, Colors,
+  Provider as PaperProvider } from 'react-native-paper';
+import { createStackNavigator, DrawerActions } from 'react-navigation';
+import { Constants } from 'expo';
+import Slideshow from 'react-native-slideshow';
+
+// Props: image URL
```

Comment:
Eventually you'll start using what's called Flow and you won't have to do this anymore, so for now, consider this kind of documentation optional.

Reply:
Just to confirm, are you referring to line 18?

Reply:
Yep.